src/addrman.cpp

```diff
@@ -626,7 +626,7 @@ void AddrManImpl::Good_(const CService& addr, bool test_before_evict, int64_t nT
 
     // if not found, bail out
     if (!pinfo)
-        return;
+        return false;
```

Comment:
Feel free to update to current project code style while touching this line:

```suggestion
    if (!pinfo) return false;
```

or

```suggestion
    if (!pinfo) {
        return false;
    }
```